client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Input, Typography } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Box } from '@mui/system';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../interface/User';
+import useStyles from './useStyles';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { useState } from 'react';
+
+interface ProfilePhotoProps {
+  header: string;
+  currentUser?: User;
+  currentProfile?: any;
+}
+
+const ProfilePhoto: React.FC<ProfilePhotoProps> = ({ header, currentUser, currentProfile }) => {
+  const classes = useStyles();
+  const [isAttached, setIsAttached] = useState(false);
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const attachFile = (event: any) => {
+    setIsAttached(true);
+    updateSnackBarMessage('File attached and ready for upload');
+  };
+
+  const uploadFile = (event: any) => {
+    // TODO: Update state and message during integration
+  };
```

Comment:
Please remove code if not used. Our code will be cleaner by removing unused and commented-out codes.